src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -1131,3 +1131,59 @@ case class TableMapGlobals(child: TableIR, newRow: IR) extends TableIR {
     TableValue(typ, newGlobals, tv.rvd)
   }
 }
+
+case class TableExplode(child: TableIR, column: String) extends TableIR {
+  def children: IndexedSeq[BaseIR] = Array(child)
+
+  private val columnIdx = child.typ.rowType.fieldIdx(column)
+  private val columnType = child.typ.rowType.types(columnIdx).asInstanceOf[TContainer]
```

Comment:
check that columnType is a container type in Python?

Reply:
Good catch, I thought that was happening.